runtime/vm/MHInterpreter.cpp

```diff
@@ -555,6 +555,26 @@ VM_MHInterpreter::dispatchLoop(j9object_t methodHandle)
 			/* Get MethodHandle for this operation from the VarHandle's handleTable */
 			j9object_t handleTable = J9VMJAVALANGINVOKEVARHANDLE_HANDLETABLE(_currentThread, varHandle);
 			j9object_t methodHandleFromTable = J9JAVAARRAYOFOBJECT_LOAD(_currentThread, handleTable, operation);
+
+			if (NULL == methodHandleFromTable) {
+				const char *exceptionClassName = "java/lang/UnsupportedOperationException";
```

Comment:
The stack needs to be walkable at this point as looking up the class may cause a GC or another exception to occur.   

This gives us two options, move the `prepareForExceptionThrow` call before looking up the exception class so we have a clean stack, or build a MT frame here.  

My preference is to build the MT frame as it will be easier to debug & service as the VarHandle will still be on the stack and findable with DDR.

Reply:
Note, we won't tear down the frame as throwing the exception will handle it appropritately

Reply:
Added `buildMethodTypeFrame`.